src/css/components/_text-track.scss

```diff
@@ -24,3 +24,7 @@
 .vjs-subtitles { color: #fff /* Subtitles are white */; }
 .vjs-captions { color: #fc6 /* Captions are yellow */; }
 .vjs-tt-cue { display: block; }
+
+video::-webkit-media-text-track-display {
+  @include transform(translateY(-45px));
```

Comment:
We should make the emulated captions and native captions work as close to each other as possible. The emulated captions already are aligned to leave room for the control bar, so, we should make sure that the native captions are similar. If the emulated captions go up/down with the control bar, we should do the same for the native captions as well.
I tried using `em` locally and it didn't actually change. But that is kind of what we want because the control bar resizes itself based on the font size :)